pkgs/development/libraries/speechd/default.nix

```diff
@@ -1,38 +1,88 @@
-{ fetchurl, lib, stdenv, intltool, libtool, pkgconfig, glib, dotconf, libsndfile
-, libao, python3Packages
-, withEspeak ? false, espeak
+{ stdenv, lib, pkgconfig, fetchurl, wrapPython
+, intltool, itstool, libtool, texinfo, autoreconfHook
+, glib, dotconf, libsndfile
+, pyxdg
+, withLibao ? true, libao
+, withPulse ? false, libpulseaudio
+, withAlsa ? false, alsaLib
+, withOss ? false
+, withFlite ? true, flite
+, withFestival ? false # TODO: , festival-freebsoft-utils
+, withEspeak-ng ? true, espeak-ng, sonic, pcaudiolib
```

Comment:
Nix calls `espeak-ng` `espeak`, the old one is `espeak-classic`. We should respect the Nix convention here.

Reply:
Done.
It makes the file contents confusing however: where to write `-ng` and where not?